web-app/uni-todo-app/src/components/TodoInputField.tsx

```diff
@@ -9,14 +9,17 @@ export const TodoInputField = () => {
   const dispatch = useAppDispatch();
 
   const addTodoEntity = useCallback(() => {
-    const todoEntity: ITodoEntity = {
-      name: input,
-      done: false,
-      createdAt: new Date().toISOString(),
-      updatedAt: new Date().toISOString(),
-    };
-    dispatch(insertTodoEntity(todoEntity));
-    setInput("");
+    if (input.trim() == "") alert("TODO field cannot be empty")
+    else {
+      const todoEntity: ITodoEntity = {
```

Comment:
I  would assign the date to a new constant and reuse it in both `createdAt` , `updatedAt`